styx-flyte-client/src/main/proto/README.md

```diff
@@ -0,0 +1,8 @@
+# Protos
+
+Styx uses these grpc and protobuf definitions to build the client and communicate with Flyte.
+
+The definitions are taken from the [flyteidl repo](https://github.com/lyft/flyteidl/tree/master/protos/flyteidl) and are built locally.
+
+## TODO
+- Add process to upgrade the protobuf definitions.
```

Comment:
It should be straight forward to put a script to pull those files (via git protocol maybe).

Reply:
👍